tests/lib/rules/indent.test.js

```diff
@@ -34,11 +34,67 @@ validExample = formatCode(
    '   var arr = [ 1, 2, 3 ],',
    '       noInit, andAgain;',
    '',
+   '   //This comment should be indented with the if statement',
    '   if (noInit) {',
    '      var a = 1,',
    '          b = 2;',
    '   }',
-   '}'
+   '}',
+   '',
+   'class MyClass {',
+   '   /**',
+   '   * This should be indented with the function.',
```

Comment:
This is supposed to be a valid example, but I think it's wrong ... isn't it supposed to be like this?

```js
class MyClass {
   /**
    * This is how it should be indented
    */
   constructor(num: number) {
      // ...
   }
}
```

Reply:
Note that the non-first lines of the block comment are indented an extra space so that the asterisks align.